pkg/ddevapp/global_dotddev_assets/commands/host/port

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+## #ddev-generated
+## Description: Helper function to launch a browser to the project URL based on the port.
+## It has 3 main uses:
+##
+## - "ddev port 3000": Open the site on port 3000.
+## - "ddev port -s 3000": Open the site, preferring a secure port.
+## - "ddev port -p 3000": Open the site, preferring an insecure (-p) port.
```

Comment:
What is a "secure" or "insecure" port? We're passing the port in, so does this actually change which port is used? Or does this just change whether `http` or `https` is used?
Either way, this needs to be clarified in the annotation.

Reply:
It is a bit confusing to me that port is an argument, but you also pass it as flags...
What happens if I go `ddev 3000 -p 8080 -s 443`?

I'd recommend:
1. removing the argument (the idea is this is used by custom commands not directly in CLI, so clarity is preferred over convenience)
1. renaming the flags to `https` and `http` with no short syntax
1. clarifying in the usage (and in docs which need to call out this new feature) what it means to "prefer" one port over the other (i.e. even without the argument, what happens with `ddev port --http=8080 --https=443`?)

Reply:
Might even make sense to only allow a single port - and then `--http` and `--https` could be booleans to indicate which protocol to use. Use https by default and if `--https` is passed, and `--http` only if that flag is passed. Fail if both are passed.